packages/modules/elasticsearch/src/elasticsearch-container.ts

```diff
@@ -1,15 +1,15 @@
 import { AbstractStartedContainer, GenericContainer, StartedTestContainer } from "testcontainers";
 
-const ELASTIC_SEARCH_HTTP_PORT = 9200;
+export const ELASTIC_SEARCH_HTTP_PORT = 9200;
 
 export class ElasticsearchContainer extends GenericContainer {
-  constructor(image = "elasticsearch:7.17.7") {
+  constructor(image = "elasticsearch:8.9.1") {
     super(image);
   }
 
   public override async start(): Promise<StartedElasticsearchContainer> {
     this.withExposedPorts(...(this.hasExposedPorts ? this.exposedPorts : [ELASTIC_SEARCH_HTTP_PORT]))
-      .withEnvironment({ "discovery.type": "single-node" })
+      .withEnvironment({ "discovery.type": "single-node", "xpack.security.enabled": "false" })
```

Comment:
I don't think it will be a good idea to default to false. In this case we can let the user decide. So, as part of the Elasticsearch container definition, this en var can be set.

Reply:
Any suggestions?

The motivation of this change was to prevent as much braking changes as possible. Without this change all users of `new ElasticsearchContainer()` (without argument passed) will get failed tests. Because I changed default value of `image` parameter.

Am I right that users can override environment by calling `.withEnvironment({ "xpack.security.enabled": "true" }`?

Reply:
When testing against specific technology, you want to tests using the right version. This is something we really encourage to do. Using default images as you mention bring the drawback you mention. But, currently the default is version 7 so if you are moving to 8 and declaring the image version then you should be aware of what implies that upgrade, in this case, security is enabled by default.

> Am I right that users can override environment by calling .withEnvironment({ "xpack.security.enabled": "true" }?

It should work.

Reply:
> you should be aware of what implies that upgrade, in this case, security is enabled by default.

Now I understand how you see the modules. It's a good idea to make them as close as possible to real-world use.

The parameter `"xpack.security.enabled": "true"` works for me because I don't care about security when testing queries/responses. **I thought testing queries/responses is the main use-case for this module.** Is there any statistic about how elastic module are used most of the time?